src/suites/cts/copyImageBitmapToTexture.spec.ts

```diff
@@ -0,0 +1,83 @@
+export const description = `
+copy imageBitmap To texture tests.
+`;
+
+import { TestGroup } from '../../framework/index.js';
+
+import { GPUTest } from './gpu_test.js';
+
+export const g = new TestGroup(GPUTest);
+
+function calculateRowPitchForImageBitmap(width: number): number {
+  if (width % 256 === 0) {
+    return 4 * width;
+  }
+  return 4 * (width + 256 - (width % 256));
+}
+
+g.test('from image element', async t => {
+  const img = document.createElement('img');
+  img.src = '../../assets/img/test.png';
+  await img.decode();
+
+  const imageBitmap = await createImageBitmap(img, 0, 0, img.width, img.height);
+  const dst = t.device.createTexture({
+    size: {
+      width: img.width,
+      height: img.height,
```

Comment:
... then use `imageBitmap.width` and `.height` here instead of `img.`

Reply:
Done.